telefunc/plugin/next/index.ts

```diff
@@ -0,0 +1,25 @@
+import type { NextConfig } from 'next'
+import { resolve } from 'path'
+
+export default plugin
+
+function plugin(nextConfig: NextConfig = {}): NextConfig {
+  return Object.assign({}, nextConfig, {
+    webpack: (config, options) => {
+      config.module.rules.push({
+        test: /\.telefunc\./,
+        use: [{ loader: resolve(__dirname, './loader.js') }],
+      })
+
+      config.module.rules.push({
+        test: /api(\/|\\)_telefunc/,
+        use: [{ loader: resolve(__dirname, './loader.js') }],
```

Comment:
@Aslemammad How about creating two loaders? One loader for two different file types is somewhat confusing.

Reply:
I'm on it, and you already did it.

Reply:
done!